bridge/client/app/_components/ktb-evaluation-details/ktb-evaluation-details.component.ts

```diff
@@ -314,11 +320,33 @@ export class KtbEvaluationDetailsComponent implements OnInit, OnDestroy {
   }
 
   _chartSeriesClicked(event) {
-    this._selectedEvaluationData = event.point.evaluationData;
+    this.selectEvaluationData(event.point.evaluationData);
   }
 
   _heatmapTileClicked(event) {
-    this._selectedEvaluationData = this._heatmapSeries[0].data[event.point.x]['evaluation'];
+    this.selectEvaluationData(this._heatmapSeries[0].data[event.point.x]['evaluation']);
+  }
+
+  selectEvaluationData(evaluation) {
+    this._selectedEvaluationData = evaluation;
+    this.highlightHeatmap();
+  }
+
+  highlightHeatmap() {
+    let highlightIndex = this._heatmapOptions.xAxis[0].categories.indexOf(moment(this._selectedEvaluationData.time).format());
+    let plotBands = [];
+    if(highlightIndex >= 0)
+      plotBands.push({
+        borderWidth: 2,
+        borderColor: '#e6be00',
+        color: 'rgba(230,190,0,.4)',
+        from: highlightIndex-0.5,
+        to: highlightIndex+0.5,
+        zIndex: 20
+      });
```

Comment:
Any chance we can use a css class for this? Or at least read the values `borderWidth`, `borderColor` and `color` from css?

Reply:
Just let me know if it's out of scope (or if we should make a refactoring task for later), but can we use `HostBinding` here?

Reply:
To read the values for `borderWidth`, `borderColor` and `color` from css, we would need to write a css-parses in ts, since the plotBands have to be provided in this config object for the heatmap and it is then drawn in a `<svg>` (the better type of `<canvas>`) and those elements inside an svg/canvas are not styled with css.

And `HostBinding` allows you to set properties on the element or component that hosts the directive, but don't see the use-case here?